_includes/v22.2/sql/range-splits.md

```diff
@@ -0,0 +1,7 @@
+CockroachDB breaks data into ranges. By default, CockroachDB attempts to keep ranges below a size of 512 MiB. To do this, the system will automatically [split a range](architecture/distribution-layer.html#range-splits) if it grows larger than this limit. For most use cases, this automatic range splitting is sufficient, and you should never need to worry about when or where the system decides to split ranges.
+
+However, there are reasons why you may want to perform manual splits on the ranges that store tables or indexes:
+
+- When a table only consists of a single range, all writes and reads to the table will be served by that range's [leaseholder](architecture/replication-layer.html#leases). If a table only holds a small amount of data but is serving a large amount of traffic, load distribution can become unbalanced. Splitting the table's ranges manually can allow the load on the table to be more evenly distributed across multiple nodes. For tables consisting of more than a few ranges, load will naturally be distributed across multiple nodes and this will not be a concern.
```

Comment:
Should this be "table or index" instead of just "table"? I see this include is referenced in alter-index.md, but after reading this, I'm not sure if it should apply to indexes, too.

Reply:
Good catch - I'm not sure and will defer to the technical review on this question!